src/main/scala/org/constellation/domain/redownload/RedownloadService.scala

```diff
@@ -453,7 +451,7 @@ class RedownloadService[F[_]: NonEmptyParallel: Applicative](
     lastMajority: SnapshotsAtHeight
   ): Long = {
     val ignorePoint = getIgnorePoint(candidateMajority.maxHeight)
-    max(joinHeight, min(max(ignorePoint, joinHeight), lastMajority.maxHeight))
+    max(joinHeight - heightInterval, min(max(ignorePoint, joinHeight), lastMajority.maxHeight))
```

Comment:
Should we handle the case of a possible negative number if `joinHeight = 0` then `cutOffHeight = -2`?

Reply:
I don't think so. Majority cannot be negative so it will neither cut nor break

Reply:
Yep, you're right.